src/api/resolvers.ts

```diff
@@ -36,8 +42,13 @@ const looInfoResolver = (property) => {
 
 const resolvers = {
   Query: {
-    loo: async (_parent, args) => await DBLoo.findById(args.id),
+    loo: async (_parent, args) => {
+      await dbConnect();
```

Comment:
here is where we shift the `dbConnect` call down into the resolver, so it isn't called until we've checked whether we have a hit in the cache store